Source/Core/Core/HW/WiimoteEmu/Extension/Drums.h

```diff
@@ -47,24 +48,28 @@ class Drums : public Extension1stParty
     // Seemingly random.
     u8 unk2 : 2;
 
-    // Always 1 with no velocity data and seemingly random otherwise.
-    u8 unk3 : 1;
+    u8 velocity3 : 1;
+
     // For which "pad" the velocity data is for.
     u8 velocity_id : 7;
 
-    // Always 1 with no velocity data and seemingly random otherwise.
-    u8 unk4 : 1;
+    u8 velocity2 : 1;
     // 1 with no velocity data and 0 when velocity data is present.
     u8 no_velocity_data_1 : 1;
     // These two bits seem to always be set. (0b11)
     u8 unk5 : 2;
     // 1 with no velocity data and 0 when velocity data is present.
     u8 no_velocity_data_2 : 1;
-    // How "soft" a drum pad has been hit as a range from 0:very-hard to 7:very-soft.
-    u8 softness : 3;
+
+    u8 velocity64 : 3;
```

Comment:
I think `velocity64` is a weird name. It makes it sound 64bits in size. Maybe just `velocity4` ?